pkg/kubelet/server/server.go

```diff
@@ -154,6 +154,10 @@ type HostInterface interface {
 	ResyncInterval() time.Duration
 	GetHostname() string
 	LatestLoopEntryTime() time.Time
+
+	// Refactor these methods out of the kubelet and move this interface out of the HostInterface and pass in separately
```

Comment:
Is this a TODO? If yes, can  you explicitly mention that? Another option is to file an issue and someone from the community might pick it up.